### PR TITLE
feat(RHTAP-974): Adding five manual CI load tests for five different component repos

### PR DESCRIPTION
**Pull request by naftalysh**
added 5 manual job configurations
into redhat-appstudio-e2e-tests-main.yaml
and after issuing "make build" command,
new 5 jobs created in redhat-appstudio-e2e-tests-main-presubmits.yaml

https://issues.redhat.com/browse/RHTAP-974

### CODE REVIEW

**Comment by naftalysh** [comment]
/pj-rehearse pull-ci-redhat-appstudio-e2e-tests-main-load-test-ci-10u-10t-go

**Comment by naftalysh** [comment]
/pj-rehearse  pull-ci-redhat-appstudio-e2e-tests-main-load-test-ci-10u-10t-nodejs

**Comment by naftalysh** [comment]
/pj-rehearse pull-ci-redhat-appstudio-e2e-tests-main-load-test-ci-10u-10t-java

**Comment by naftalysh** [comment]
/pj-rehearse  pull-ci-redhat-appstudio-e2e-tests-main-load-test-ci-10u-10t-python

**Comment by naftalysh** [comment]
/pj-rehearse  pull-ci-redhat-appstudio-e2e-tests-main-load-test-ci-10u-10t-quarkus

File: ci-operator/config/redhat-appstudio/e2e-tests/redhat-appstudio-e2e-tests-main.yaml [hunk 1]
```diff
@@ -87,6 +87,46 @@ tests:
...
+    workflow: redhat-appstudio-load-test
+  timeout: 8h0m0s
+- always_run: false
+  as: load-test-ci-10u-10t-quarkus
+  steps:
+    cluster_profile: aws
+    env:
+      SCENARIO: ci-10u-10t-quarkus
+    workflow: redhat-appstudio-load-test
+  timeout: 8h0m0s
```
Comment (pmacik) [inline ci-operator/config/redhat-appstudio/e2e-tests/redhat-appstudio-e2e-tests-main.yaml:129]: @naftalysh This job is effective duplicate of the `load-test-ci-10u-10t` job

File: ci-operator/config/redhat-appstudio/e2e-tests/redhat-appstudio-e2e-tests-main.yaml [hunk 2]
```diff
@@ -87,6 +87,46 @@ tests:
...
+- always_run: false
+  as: load-test-ci-10u-10t-python
+  steps:
+    cluster_profile: aws
+    env:
+      SCENARIO: ci-10u-10t-python
+    workflow: redhat-appstudio-load-test
+  timeout: 8h0m0s
+- always_run: false
+  as: load-test-ci-10u-10t-quarkus
```
Comment (pmacik) [inline ci-operator/config/redhat-appstudio/e2e-tests/redhat-appstudio-e2e-tests-main.yaml:123]: @naftalysh This `load-test-ci-10u-10t-quarkus` job is effective duplicate of the `load-test-ci-10u-10t` job

**Comment by naftalysh** [comment]
/pj-rehearse pull-ci-redhat-appstudio-e2e-tests-main-load-test-ci-10u-10t-python

**Comment by naftalysh** [comment]
/pj-rehearse pull-ci-redhat-appstudio-e2e-tests-main-load-test-ci-10u-10t-python

**Comment by pmacik** [comment]
@naftaly Try to rebase on top of the latest `master` and re-run the `make update`

**Pull request merged**